Header header
shape_msgs/Plane[] planes
sensor_msgs/PointCloud2[] clouds
sensor_msgs/PointCloud2 point_cloud